### ISSUE TITLE
Delete / make events invisible past expiration date

### ISSUE THREAD

**Issue by jonathan-s** [thepracticaldev/dev.to#3056]
**Is your feature request related to a problem? Please describe.**

An event takes place in time, so after the event date it would be prudent to make it invisible. Or alternatively select a button to browse past events. 

See this event for instance: https://dev.to/listings/events/webinar-cracking-the-javascript-coding-interview-1f7d 

It is no longer relevant.

**Comment by Zhao-Andy** [comment]
Fair point, thanks for the heads up! I updated your issue title to make a bit more sense for the team, since we used to have native `events` on the site.

**Comment by jonathan-s** [comment]
👍
